nix/modules/holochain.nix

```diff
@@ -55,14 +55,16 @@
       });
 
       # derivation with the main crates
-      holochain = craneLib.buildPackage (commonArgs // {
+      holochain = lib.makeOverridable craneLib.buildPackage (commonArgs // {
         CARGO_PROFILE = "release";
         cargoArtifacts = holochainDepsRelease;
         src = flake.config.srcCleanedHolochain;
         doCheck = false;
         passthru.src.rev = flake.config.reconciledInputs.holochain.rev;
       });
 
+      holochain_chc = holochain.override { cargoExtraArgs = " --features chc"; };
```

Comment:
Does this mean we'll always build `holochain `+ `chc` on CI as a different build of the `holochain` binary? If so I'd prefer this to be a doc example.

I'm a fan of testing features in general but I think it's too slow on PR runs

Reply:
Right, and if that's so, what about cachix?

Reply:
we don't build this on CI automatically. 

it's exposed for whoever wants it and it'll require compiling.